app/pages/collections/list.cjsx

```diff
@@ -24,33 +38,73 @@ List = React.createClass
-    "/collections/#{collection_owner}/#{collection_name}"
+    baseLink = "/"
+    if @props.project?
+      baseLink += "projects/#{@props.project.slug}/"
+    "#{baseLink}collections/#{collection.slug}"
 
   listCollections: ->
     query = {}
     if @props.params?.collection_owner?
       query.owner = @props.params.collection_owner
-      query.include = 'owner'
-
+    else if @props.params?.profile_name?
+      query.owner = @props.params.profile_name
+    if @props.project?
+      query.project_ids = @props.project.id
     query.favorite = @props.favorite
     Object.assign query, @props.location.query
 
     apiClient.type('collections').get query
 
   render: ->
     {location} = @props
-    <section className="resources-container collections-container">
+    classes = classNames {
+      "resources-container": true
+      "collections-container": true
+      "in-project-context": @props.project?
+    }
+
+    baseLink = if @props.project? then "/projects/#{@props.project.slug}/" else "/"
+    baseType = if @props.favorite then "favorites" else "collections"
+    shouldShowMoreLink = @props.params.profile_name? || @props.params.collection_owner?
+    showMoreLink = "#{baseLink}#{baseType}/"
+    shouldShowSubMenu = @props.project? && !@props.params.profile_name?
+    shouldShowZooWideLink = @props.project? && !shouldShowMoreLink
+    showZooWideLink = "/#{baseType}/"
+    shouldShowUserWideLink = @props.project? && shouldShowMoreLink
+    userForUserWideLink = if @props.params.profile_name? then @props.params.profile_name else @props.params.collection_owner
+    showUserWideLink = "/users/#{userForUserWideLink}/#{baseType}"
+
+    <section className={classes}>
+      {if shouldShowSubMenu
+        <CollectionsNav
+          translationObjectName="#{@props.translationObjectName}"
+          user={@props.user}
+          project={@props.project} />}
       <PromiseRenderer promise={@listCollections()}>{(collections) =>
         if collections?.length > 0
           meta = collections[0].getMeta()
           <div>
             <div className="resource-results-counter collection-results-counter">
-              {if meta
-                pageStart = meta.page * meta.page_size - meta.page_size + 1
-                pageEnd = Math.min(meta.page * meta.page_size, meta.count)
-                count = meta.count
-                <Translate pageStart={pageStart} pageEnd={pageEnd} count={count} content="#{@props.translationObjectName}.countMessage" component="p" />}
+              <p>
+                {if meta
+                  pageStart = meta.page * meta.page_size - meta.page_size + 1
+                  pageEnd = Math.min(meta.page * meta.page_size, meta.count)
+                  count = meta.count
+                  <Translate pageStart={pageStart} pageEnd={pageEnd} count={count} content="#{@props.translationObjectName}.countMessage" />}
+                {if shouldShowMoreLink
+                  <Link className="show-more" to="#{showMoreLink}">
```

Comment:
`showMoreLink` doesn't need to be string interpolated. It's already a string. Just set it as `to={showMoreLink}`